lib/practice_exercises.rb

```diff
@@ -1,13 +1,34 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  list.each do |i|
+    if list[i] == list[i + 1]
+      list.delete_at(i)
+    end
+  end
+  return list
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
```

Comment:
If the words are small this is O(n), otherwise I would say the time complexity is O(mn) where m is the length of the words.